app/assets/src/api/amr.js

```diff
@@ -1,6 +1,6 @@
 import { get } from "./core";
 
-export const getAMRCounts = sampleIds =>
```

Comment:
Don't think you need the `async` here, since you don't have an `await`.

Another property of async functions is they will always return a promise, and will wrap a non-promise return value (e.g. `return {}`) in a promise. Since you're returning a promise with `get()` I don't think the `async` makes any difference.